php/Blocks/Controls/InnerBlocks.php

```diff
@@ -53,7 +53,7 @@ public function validate( $value, $echo ) {
 		$content = genesis_custom_blocks()->loader->get_data( 'content' );
 
 		return empty( $content )
-			? urldecode( filter_input( INPUT_GET, 'inner_blocks' ) )
+			? urldecode( wp_strip_all_tags( filter_input( INPUT_GET, 'inner_blocks' ) ) )
```

Comment:
`wp_strip_all_tags()` is similar to what the `FILTER_SANITIZE_STRING` argument did before